iterators.tex

```diff
@@ -6442,56 +6461,84 @@
 container copies the elements, which cannot be done in constant time.
 \exitexample
 
-\begin{codeblock}
+\begin{itemdecl}
 @\newtxt{template <class T>}@
-@\newtxt{struct is_view;}@
+@\newtxt{struct enable_view \{ \};}@
 
 @\newtxt{struct view_base \{ \};}@
 
+@\newtxt{template <class T>}@
+@\newtxt{constexpr bool \xname{view_predicate}; // \expos}@
```

Comment:
Is this valid C++? I think `constexpr` objects need to be initialized. Might need an `= /seebelow;`.